nixos/doc/manual/from_md/installation/building-nixos.chapter.xml

```diff
@@ -34,7 +34,9 @@
   <section xml:id="sec-building-image-instructions">
     <title>Practical Instructions</title>
     <programlisting>
-$ git clone https://github.com/NixOS/nixpkgs.git
+# a depth of 1 will result in a much faster download and setup of nixpkgs
```

Comment:
```suggestion
# The `depth` parameter in Git is used when you want to create a shallow clone of a repository. Shallow clone means downloading less history from the repository, thereby reducing the amount of data that needs to be transferred, which can significantly speed up the cloning process.
# For example, if you use git clone --depth 1, you're asking Git to only pull down one commit worth of historical data. This means you get the latest version of the codebase, but none of the history, which can be useful in cases where the history is large and you don't need it. It's quicker because you're simply downloading less data.
# However, note that this is a trade-off. While you gain speed and save disk space, you lose the repository's history and some functionalities like viewing log history beyond the shallow depth, searching for changes, and certain kinds of advanced Git operations.
```